daemon/retake_failed_swap_test.go

```diff
@@ -0,0 +1 @@
+package daemon
```

Comment:
why is this file empty?

Reply:
Sorry, didn't mean for that file to make it into the PR. Removed. it.